### ISSUE TITLE
A pet AI problem

### ISSUE THREAD

**Issue by Tatado2020** [xXKeyleXx/MyPet#1378]
Hello, Mypet plug-in is really a very good plug-in, but this plug-in used in my server has a problem: the AI of the pet does not work properly, it acts like it is playing PPT when following the player, but the other biological AI of the server are normal, only the pet is such a situation, do you know what this may be caused?

**Comment by xXKeyleXx** [comment]
Did you change any mob related settings in the spigot.yml/bukkit.yml?

**Comment by Tatado2020** [comment]
> 您是否在spigot.yml / bukkit.yml中更改了与生物有关的任何设置？

 Yes, I adjusted it before. Can I ask what settings will affect pet AI? All I can think of at the moment is [entity-activation-range] this one, but I have adjusted the value of this item to 32

**Comment by Tatado2020** [comment]
> Did you change any mob related settings in the spigot.yml/bukkit.yml?

Hello, I finally found the source of the pet AI problem: I modified the misc in the entity-activation-range in the spigot.yml file to 2 before, so it caused the player to cause the pet AI activity not to be counted as long as more than 2 squares away from the pet, and now I have changed the misc back to 16 and the pet AI is back to normal.
